### ISSUE TITLE
Using segvol model

### ISSUE THREAD

**Issue by 2384441037** [BAAI-DCAI/SegVol#34]
If I use the model to inference， the gt_path is must needed? If i only have the CT image, how do I do next to aviod using gt_path?

**Comment by 2384441037** [comment]
I meet anthoer question, when I use the inference_demo.sh, how can I save the logits_global_single to nii.gz just like this 

![Image](https://github.com/user-attachments/assets/e64988a8-53d2-456a-9580-bd2f7a4d6ec1)

I change codes in inference_demo.py, add 

def save_preds(ct_path, save_path, logits_mask, start_coord, end_coord):
        ct = nib.load(ct_path)
        logits_mask = logits_mask.transpose(-1, -3)
        start_coord[-1], start_coord[-3] = start_coord[-3], start_coord[-1]
        end_coord[-1], end_coord[-3] = end_coord[-3], end_coord[-1]
        preds_save = torch.zeros(ct.shape)
        preds_save[start_coord[0]:end_coord[0], 
                        start_coord[1]:end_coord[1], 
                        start_coord[2]:end_coord[2]] = torch.sigmoid(logits_mask)
        preds_save = torch.where(preds_save > 0.5, 1., 0.).numpy()
        preds_nii = nib.Nifti1Image(preds_save, affine=ct.affine, header=ct.header)
        nib.save(preds_nii, save_path)
        print(f'-----------------{save_path}保存成功-----------------------') 

but only liver and spleen sucesse,

![Image](https://github.com/user-attachments/assets/2da7bad0-8b63-45fa-8878-4aceb3f5aa35)
